plugins/transforms/tableinput/src/main/java/org/apache/hop/pipeline/transforms/tableinput/TableInputDialog.java

```diff
@@ -377,22 +388,27 @@ private List<String> getSqlReservedWords() {
       databaseMetaData = db.getDatabaseMetaData();
       if (databaseMetaData == null) {
         logError("Couldn't get database metadata");
-        return new ArrayList<>();
+        return;
       }
       List<String> sqlKeywords = new ArrayList<>();
       try {
         final ResultSet functionsResultSet = databaseMetaData.getFunctions(null, null, null);
         while (functionsResultSet.next()) {
-          sqlKeywords.add(functionsResultSet.getString("FUNCTION_NAME"));
+          String functionName = functionsResultSet.getString("FUNCTION_NAME");
+          if (functionName.contains(";")) {
+            functionName = functionName.substring(0, functionName.indexOf(";"));
```

Comment:
this string replacement is here because of MS SQL Server: in fact, the `getFunctions()` method returns functions with something more. Like, if the function name is `helloworld()`, it returns `helloworld;1`, so we have to remove the substring from the semicolon on